01-lesson_notebooks/01b-How_to_Use_the_Class_Materials.ipynb

```diff
@@ -41,16 +41,11 @@
     "Once you have completed your readings and you feel comfortable applying the lessons, you can begin the IDE project(s) associated with that particular week.\n",
     "Open your Visual Studio Code IDE application. Then navigate to the file tab at the top and select open folder. Then it is just a matter of navigating to the corresponding IDE project folder you’d like to work on.\n",
     "\n",
-    "You can start at any parent folder, but to prevent an eye-sore, I’m going to navigate down to project-2a as an example:\n",
-    "\n",
-    "Now your editor should look something like this:\n",
```

Comment:
we can't insert images yet, the reader doesn't have the markdown plugin installed yet to be able to view them

Reply:
I'm able to see them on GitHub and locally. Are you saying you cannot see them?

Reply:
1. I don't see any images on github or locally, only a blank space where the image should be, I'm using stock vscode/codium with the recommended jupyter package
2. yes the screenshot was intentional, I made one because I couldn't find one

Reply:
hmm thanks for checking for this. I'll create a ticket to investigate.

Reply:
As we discussed off-github, image display is working, but this section doesn't have images yet